### PR TITLE
Wishlist page and Cart page 

### PR DESCRIPTION
**Pull request by rahul0396**
https://user-images.githubusercontent.com/43301962/154841304-e68dc8bc-9a30-4410-af2e-a09bce9b1f18.mp4

### CODE REVIEW

File: css/cartpage.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
+@import url("/css/navbar.css");
+.my-cart{
+    position: absolute;
+width: 100%;
```
Comment (RakshithBhat03) [inline css/cartpage.css:4]: You can indent these lines. Use a code formatter

File: css/cartpage.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+@import url("/css/navbar.css");
+.my-cart{
+    position: absolute;
+width: 100%;
+height: 1000px;
```
Comment (RakshithBhat03) [inline css/cartpage.css:5]: Use `rem`  or `vh` unit here.

File: css/cartpage.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
+@import url("/css/navbar.css");
+.my-cart{
+    position: absolute;
+width: 100%;
+height: 1000px;
+left: 0px;
+top: 80px;
+
+background: #F7F7F7;
```
Comment (RakshithBhat03) [inline css/cartpage.css:9]: You can use a variable here.

File: css/cartpage.css [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    
+}
+
+
+
+
+/* price details */
+.price-details{
+    position: absolute;
+width: 520px;
```
Comment (RakshithBhat03) [inline css/cartpage.css:52]: Again, indent the code and use `rem` unit wherever possible

File: pages/cartpage.html [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+       
+       <div class="product-card">
+           <img class="imga" src="/assets/image/apple cider vinegar round 1.jpg" alt="">
+           <div class="card-data">
+               <h1>Apple cider vinegar</h1>
+               <div class="card-data-price">
+
+                   <p >₹ 299</p>
+                   <!-- <p >₹ </p> -->
```
Comment (RakshithBhat03) [inline pages/cartpage.html:63]: You can remove these comments

File: wishlist.css [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    width: 100%;
+    height: 100%;
+}
+.wishlist-heading{
+    margin: 1rem;
+    
+    text-align: center;
+}
+.fa-heart{
+    background-color: rgb(216, 44, 44);
```
Comment (RakshithBhat03) [inline wishlist.css:15]: Use variables here as well

File: pages/cartpage.html [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+    </nav>
+
+   </div> 
+   <!-- navbar end -->
+   <!-- product card landscape start  -->
+   <div class="my-cart">
+
+       
+       <div class="product-card">
+           <img class="imga" src="/assets/image/apple cider vinegar round 1.jpg" alt="">
```
Comment (kavyakunder) [inline pages/cartpage.html:57]: add the description of the image in alt

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested a few changes. You can merge once implemented.

**Review by kavyakunder — APPROVED** [review]
Use prettier to format the code

**Pull request merged**